pkg/config/kubeClient_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package config
+
+import (
+
+    "reflect"
+	"testing"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/rest"
+)
+
+// should build config from flags if kube config is not empty
+func Test_GetConfig(t *testing.T) {
+
+	clientConfig := GetConfig()
+	fromFlags, _ := clientcmd.BuildConfigFromFlags("", Cfg.KubeConfig)
```

Comment:
Please add check to see if Cfg.KubeConfig is not empty before.

Reply:
added check